deploy/olm-catalog/ibm-iam-operator/3.6.1/operator.ibm.com_authentications_crd.yaml

```diff
@@ -0,0 +1,236 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: authentications.operator.ibm.com
+  labels:
+    app.kubernetes.io/instance: ibm-iam-operator
+    app.kubernetes.io/managed-by: ibm-iam-operator
+    app.kubernetes.io/name: ibm-iam-operator
+spec:
+  group: operator.ibm.com
+  names:
+    kind: Authentication
+    listKind: AuthenticationList
+    plural: authentications
+    singular: authentication
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: Authentication is the Schema for the authentications API
```

Comment:
should `Schema` be lower case?

Reply:
Yeah `Schema` can be lowercase. We've had it uppercase for the past two csv's.